networking/enable-cluster-wide-proxy.adoc

```diff
@@ -14,7 +14,7 @@ The cluster-wide proxy is only supported if you used a user-provisioned infrastr
 
 .Prerequisites
 
-* Review the  xref:../installing/install_config/configuring-firewall.adoc#configuring-firewall[sites that your cluster requires access to] and determine whether any need to bypass the proxy. By default, all cluster egress traffic is proxied, including calls to hosting cloud provider APIs. Add sites to the Proxy object's `spec.noProxy` field to bypass the proxy if necessary.
+* Review the  xref:../installing/install_config/configuring-firewall.adoc#configuring-firewall[sites that your cluster requires access to] and determine whether any need to bypass the proxy. By default, all cluster egress traffic is proxied, including calls to the cloud provider API for the cloud that hosts your cluster. Add sites to the Proxy object's `spec.noProxy` field to bypass the proxy if necessary.
```

Comment:
s/any need to/any of them need to/ (for clarity)

Reply:
> ! need to v
Avoid using “need to” to refer to a requirement that a product or procedure imposes.

Thanks for pointing that out! I'll pick a different clarification.